media-gfx/drawpile/drawpile-2.0.10.recipe

```diff
@@ -66,6 +67,8 @@ INSTALL()
 	mkdir -p $appsDir/Drawpile
 	cp ./build/bin/drawpile $appsDir/Drawpile/Drawpile
 	cp ./build/bin/drawpile-srv $appsDir/Drawpile/drawpile-srv
+	addResourcesToBinaries $portDir/additional-files/Drawpile.rdef \
+		$appsDir/Drawpile/Drawpile
 	addAppDeskbarSymlink $appsDir/Drawpile/Drawpile
 }
 
```

Comment:
I did not try the recipe yet but it looks OK.
Oops, you can remove this line:
```diff
-# TODO Add a rdef icon for Drawpile
```
as well as the last one, which is empty.

Hmm, if you're using PE or if you're editing the file on a popular and commercial OS then it's better to not remove the last line, because it will be worse: some editors remove the last LineFeed if the last line is not empty. vim does not, so you can get rid of this empty line if you use vim.

Reply:
Done!
Sorry for forgetting about the comment... I was in a rush :o 
Should I submit my task?